osh/builtin_assign.py

```diff
@@ -247,7 +248,7 @@ def Run(self, cmd_val):
       for pair in cmd_val.pairs:
         if pair.rval is not None:
           e_usage("doesn't accept RHS with -n",
-                  span_id=word_.LeftMostSpanForWord(pair.blame_word))
+                  loc.Span(word_.LeftMostSpanForWord(pair.blame_word)))
```

Comment:
loc.Word(pair.blame_word) should work here

The exception handler should be smart enough (and we should be able to improve it )